specification/cognitiveservices/OpenAI.Inference/models/completions/chat_completions.tsp

```diff
@@ -3,75 +3,37 @@ import "@typespec/http";
 
 namespace Azure.OpenAI;
 
-@doc("A description of the intended purpose of a message within a chat completions interaction.")
-enum ChatRole {
-  @doc("The role that instructs or sets the behavior of the assistant.")
-  system: "system",
-
-  @doc("The role that provides responses to system-instructed, user-prompted input.")
-  assistant: "assistant",
-
-  @doc("The role that provides input for chat completions.")
-  user: "user",
-
-  @doc("The role that provides function results for chat completions.")
-  @added(ServiceApiVersions.v2023_07_01_Preview)
-  function: "function",
-
-  @doc("The role that represents extension tool activity within a chat completions operation.")
-  @added(ServiceApiVersions.v2023_08_01_Preview)
-  tool: "tool",
-}
-
-@doc("A single, role-attributed message within a chat completion interaction.")
-model ChatMessage {
-  @doc("The role associated with this message payload.")
-  @projectedName("json", "role")
-  role: ChatRole;
-
-  #suppress "@azure-tools/typespec-azure-core/no-nullable" "we explicitly want a nullable string"
-  @doc("The text associated with this message payload.")
-  @projectedName("json", "content")
-  content: string | null;
-
-  @doc("""
-    The name of the author of this message. `name` is required if role is `function`, and it should be the name of the
-    function whose response is in the `content`. May contain a-z, A-Z, 0-9, and underscores, with a maximum length of
-    64 characters.
-    """)
-  @added(ServiceApiVersions.v2023_07_01_Preview)
-  name?: string;
-
-  @doc("The name and arguments of a function that should be called, as generated by the model.")
-  @added(ServiceApiVersions.v2023_07_01_Preview)
-  @projectedName("json", "function_call")
-  functionCall?: FunctionCall;
+@added(ServiceApiVersions.v2023_12_01_Preview)
+@doc("The valid response formats Chat Completions can provide. Used to enable JSON mode.")
+enum ChatCompletionsResponseFormat {
+  @doc("Use the default, plain text response format.")
+  text: "text",
 
   @doc("""
-    Additional context data associated with a chat message when requesting chat completions using compatible Azure
-    OpenAI chat extensions. This includes information like the intermediate data source retrievals used to service a
-    request.
-    This context information is only populated when using Azure OpenAI with chat extensions capabilities configured.
+  Use a response format that guarantees emission of a valid JSON object. Only structure is guaranteed and contents must
+  still be validated.
   """)
-  @added(ServiceApiVersions.v2023_08_01_Preview)
-  @projectedName("csharp", "AzureExtensionsContext")
-  context?: AzureChatExtensionsMessageContext;
+  jsonObject: "json_object",
 }
 
 @doc("""
 The configuration information for a chat completions request.
 Completions support a wide variety of tasks and generate text that continues from or "completes"
 provided prompt data.
 """)
+@added(ServiceApiVersions.v2023_05_15)
```

Comment:
Oh, I see now why we had an name update in the `data-plane/stable` folder for that service release date. Is this on purpose or maybe a pasting mistaking?

Reply:
That's now our first supported label with chat completions support and adding newly-versioned children items into the unversioned parent was causing issues. I believe the 5-15 backdating is accurate.